packages/main/src/TimePicker.js

```diff
@@ -0,0 +1,640 @@
+		 *
+		 * @type {string}
+		 * @defaultvalue "None"
+		 * @public
+		 */
+		valueState: {
+			type: ValueState,
+			defaultValue: ValueState.None,
+		},
+
+		/**
+		 * Determines whether the <code>ui5-timepicker</code> is displayed as disabled.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+
+		/**
+		 * Determines whether the <code>ui5-timepicker</code> is displayed as readonly.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		readonly: {
+			type: Boolean,
+		},
+
+		_isPickerOpen: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_respPopover: {
+			type: Object,
+		},
+
+		_hours: {
+			type: String,
+		},
+
+		_minutes: {
+			type: String,
+		},
+
+		_seconds: {
+			type: String,
+		},
+
+		isTwelveHoursFormat: {
+			type: Boolean,
+		},
+	},
+	slots: /** @lends sap.ui.webcomponents.main.TimePicker.prototype */ {
+		//
+	},
+	events: /** @lends sap.ui.webcomponents.main.TimePicker.prototype */ {
+		/**
+		 * Fired when the input operation has finished by pressing Enter or on focusout.
+		 *
+		 * @event
+		 * @public
+		*/
+		change: {},
+		/**
+		 * Fired when the value of the <code>ui5-timepicker</code> is changed at each key stroke.
+		 *
+		 * @event
+		 * @public
+		*/
+		input: {},
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ * The <code>ui5-timepicker</code> component provides an input field with assigned sliders which opens on user action.
+ * The <code>ui5-timepicker</code> allows users to select a localized time using touch,
+ * mouse, or keyboard input. It consists of two parts: the time input field and the
+ * sliders.
+ *
+ * <h3>Usage</h3>
+ * The user can enter a time by:
+ * <ul>
+ * <li>Using the sliders that opens in a popup</li>
+ * <li>Typing it in directly in the input field</li>
+ * </ul>
+ * <br><br>
+ * When the user makes an entry and chooses the enter key, the sliders shows the corresponding time.
+ * When the user directly triggers the sliders display, the actual time is displayed.
+ * For the <code>ui5-timepicker</code>
+ *
+ * <h3>Formatting</h3>
+ *
+ * If a time is entered by typing it into
+ * the input field, it must fit to the used time format.
+ * <br><br>
+ * Supported format options are pattern-based on Unicode LDML Date Format notation.
+ * For more information, see <ui5-link target="_blank" href="http://unicode.org/reports/tr35/#Date_Field_Symbol_Table" class="api-table-content-cell-link">UTS #35: Unicode Locale Data Markup Language</ui5-link>.
+ * <br><br>
+ * For example, if the <code>format-pattern</code> is "HH:mm:ss",
+ * a valid value string is "11:42:35" and the same is displayed in the input.
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import @ui5/webcomponents/dist/TimePicker.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.TimePicker
+ * @extends UI5Element
+ * @tagname ui5-timepicker
```

Comment:
add @ since 1.0.0-rc.6 (without the space between @ and since)

Reply:
Done